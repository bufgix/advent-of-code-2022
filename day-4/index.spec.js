import { describe, test, expect } from "vitest";

import { solution1, solution2 } from "./index.js";
import { input } from "./input";

const testInput = `2-4,6-8
2-3,4-5
5-7,7-9
2-8,3-7
6-6,4-6
2-6,4-8`;

describe("day 3", () => {
  test("q1", () => {
    expect(solution1(testInput)).toBe(2);
    expect(solution1(input)).toMatchSnapshot();
  });

  test("q1", () => {
    expect(solution2(testInput)).toBe(4);
    expect(solution2(input)).toMatchSnapshot();
  });
});
