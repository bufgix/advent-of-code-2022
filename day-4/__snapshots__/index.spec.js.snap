// Vitest Snapshot v1

exports[`day 3 > q1 1`] = `496`;

exports[`day 3 > q1 2`] = `847`;
